src/ui/popups/info.js

```diff
@@ -113,6 +113,11 @@ require([
 		$('#album-art').css('background-image', `url("${getCoverArt()}")`);
 	}
 
+	function fillUserPlayCount() {
+		$('#userPlayCountLabel').text(`${i18n('infoYourScrobbles')}:`);
```

Comment:
Also, you could use string template in locales:
```json
  "infoYourScrobbles": {
    "message": "Your scrobbles: $1",
    "description": "Label for user play count"
  },
```

```js
$('#userPlayCountLabel').text(i18n('infoYourScrobbles', getUserPlayCount()));
```

Reply:
> Also, you could use string template in locales:
> 

Thanks for pointing that out. I deliberately kept the i18n string simple to potentially make it more convenient for translators to work on it. Perhaps it's just me who feels that way?

Reply:
Probably, no. As for me, I think templates help translators to understand better how exactly a string is used in the extension.

Anyway, you can leave the string as is.